raid.go

```diff
@@ -20,20 +14,30 @@ type Raid struct {
 	State   string
 	Devices []string
 	Failed  []int
+	Active  []int
 }
 
 var failed = regexp.MustCompile("\\[([U_]+)\\]")
 
-func (r Raid) GetFailedPhysicalDevices() []string {
-	devices := []string{}
+func (r Raid) GetFailedAndMissingPhysicalDevices() (failedDevices []string, missingDevicesCount int) {
 	for _, fd := range r.Failed {
```

Comment:
what does `fd` stand for?
Maybe try to use more explicit variable names. Sometimes the short forms are obvious but I'm not sure in this case

Reply:
changed